test/parallel/test-fs-utils-get-dirents.js

```diff
@@ -88,24 +89,27 @@ const filename = 'foo';
     common.mustCall((err, dirent) => {
       assert.strictEqual(err, null);
       assert.strictEqual(dirent.name, filenameBuffer);
+      assert.strictEqual(dirent.parentPath, tmpdir.path);
       common.expectWarning(
         'DeprecationWarning',
         'dirent.path is deprecated in favor of dirent.parentPath',
         'DEP0178');
-      assert.deepStrictEqual(dirent.path, Buffer.from(tmpdir.resolve(`${filename}/`)));
+      assert.strictEqual(dirent.path, tmpdir.path);
```

Comment:
That's a breaking change, it doesn't make sense to introduce a breaking change to a deprecated feature